test/parallel/test-tls-no-cert-required.js

```diff
@@ -1,4 +1,5 @@
 'use strict';
+var assert = require('assert');
```

Comment:
Nit: var->const in this file too if you are so inclined (and not if you are not).